build/teamcity-release-notes.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+cockroach_ref=$(git describe --tags --exact-match 2>/dev/null || git rev-parse HEAD)
+
+# A merge may contain multiple PRs due to craig(bot) activity
+# Get PR numbers for current merge
+curl -H 'Authorization: token $GITHUB_API_TOKEN' https://api.github.com/search/issues?q=sha:$cockroach_ref+repo:cockroachdb/cockroach | jq .items[].number > prnums
+prcount=$(cat prnums | wc -l)
+
+# For each PR, process its commit(s)
+process_pr_commits() {
+  # Get commits given PR number
+  commits=$(curl -H 'Authorization: token $GITHUB_API_TOKEN' https://api.github.com/repos/cockroachdb/cockroach/pulls/$1/commits)
+  commits="${commits//$'\''/'%27'}"
+  echo $commits > commits.json
+  pr_title=$(curl -H 'Authorization: token $GITHUB_API_TOKEN' https://api.github.com/repos/cockroachdb/cockroach/pulls/$1 | jq .title)
+  commitcount=$( jq length commits.json )
+  commitctr=0
+
+  # Iterate through all commit messages on this PR
+  while [ $commitctr -lt $commitcount ]
+  do
+    jq -r --arg i "$commitctr" '.[$i|tonumber].commit.message' commits.json > message
+    # For each commit message, check whether it contains at least one release note
+    bool=$( jq -r --arg i "$commitctr" '.[$i|tonumber].commit.message|test("[rR]elease [nN]ote")' commits.json )
+    # Extract the type(s) of release note(s)
+    jq -r --arg i "$commitctr" '.[$i|tonumber].commit.message|match("[rR]elease [nN]ote (?<type>.*): .*")' commits.json > match.json
+    # Check whether there exists at least one release note type that is not
+    # "none," "bug"-, or "performance"-related
+    jq -r '.captures | .[] | .string|test("[nN]one|[bB]ug|[pP]erformance")' match.json > valid
```

Comment:
Why should bug and performance messages be skipped?

Reply:
@jseldess, could you provide the context here?